GUI/Controls/ModInfoTabs/Metadata.Designer.cs

```diff
@@ -66,15 +68,17 @@ private void InitializeComponent()
             this.MetadataTable.Controls.Add(this.LicenseLabel, 0, 1);
             this.MetadataTable.Controls.Add(this.MetadataModuleLicenseTextBox, 1, 1);
             this.MetadataTable.Controls.Add(this.AuthorLabel, 0, 2);
+            this.MetadataTable.Controls.Add(this.DownloadCountLabel, 0, 3);
+            this.MetadataTable.Controls.Add(this.DownloadCountTextBox, 1, 3);
             this.MetadataTable.Controls.Add(this.AuthorsPanel, 1, 2);
-            this.MetadataTable.Controls.Add(this.ReleaseLabel, 0, 3);
-            this.MetadataTable.Controls.Add(this.MetadataModuleReleaseStatusTextBox, 1, 3);
-            this.MetadataTable.Controls.Add(this.GameCompatibilityLabel, 0, 4);
-            this.MetadataTable.Controls.Add(this.MetadataModuleGameCompatibilityTextBox, 1, 4);
-            this.MetadataTable.Controls.Add(this.IdentifierLabel, 0, 5);
-            this.MetadataTable.Controls.Add(this.MetadataIdentifierTextBox, 1, 5);
-            this.MetadataTable.Controls.Add(this.ReplacementLabel, 0, 6);
-            this.MetadataTable.Controls.Add(this.ReplacementTextBox, 1, 6);
+            this.MetadataTable.Controls.Add(this.ReleaseLabel, 0, 4);
+            this.MetadataTable.Controls.Add(this.MetadataModuleReleaseStatusTextBox, 1, 4);
+            this.MetadataTable.Controls.Add(this.GameCompatibilityLabel, 0, 5);
+            this.MetadataTable.Controls.Add(this.MetadataModuleGameCompatibilityTextBox, 1, 5);
+            this.MetadataTable.Controls.Add(this.IdentifierLabel, 0, 6);
+            this.MetadataTable.Controls.Add(this.MetadataIdentifierTextBox, 1, 6);
+            this.MetadataTable.Controls.Add(this.ReplacementLabel, 0, 7);
+            this.MetadataTable.Controls.Add(this.ReplacementTextBox, 1, 7;
```

Comment:
A close parenthesis was lost here:

```suggestion
            this.MetadataTable.Controls.Add(this.ReplacementTextBox, 1, 7);
```

Reply:
Yes, I saw that. Has been fixed :)